docs/src/AlgebraicGeometry/Schemes/AffineSchemes.md

```diff
@@ -32,12 +32,7 @@ Spec(R::MPolyRing, I::MPolyIdeal)
 Spec(R::MPolyRing, U::AbsMPolyMultSet)
 Spec(R::MPolyRing, I::MPolyIdeal, U::AbsMPolyMultSet)
 ```
-
-### Copy constructors
-
-```@docs
-Spec(X::Spec)
-```
+See [`inclusion_morphism(::AbsSpec, ::AbsSpec)`](@ref) for a way to obtain the ideal ``I`` from ``X = \mathrm{Spec}(R, I)``.
```

Comment:
Maybe if you try this instead?
```suggestion
See [`inclusion_morphism(X::AbsSpec, Y::AbsSpec; check::Bool)`](@ref) for a way to obtain the ideal ``I`` from ``X = \mathrm{Spec}(R, I)``.
```
or perhaps
```suggestion
See [`inclusion_morphism(X::AbsSpec, Y::AbsSpec; check::Bool=true)`](@ref) for a way to obtain the ideal ``I`` from ``X = \mathrm{Spec}(R, I)``.
```

The other option would be to try and use a [Named doc `@ref`](https://documenter.juliadocs.org/stable/man/syntax/#Named-doc-@refs)

Finally, I would suggest to ask in the `#documentation` channel of the Julia Slack (not the OSCAR slack!) for hints on how to do this -- we might as well have hit on a bug in Documenter.jl here (wouldn't be the first time). The Documenter.jl documentation at times could be ... more precise and detailed and complete ... (just like ours ;-) )